apps/mudita-center-e2e/src/specs/overview/kompakt-overview.ts

```diff
@@ -61,55 +61,68 @@ describe("E2E mock sample - overview view", () => {
     )
 
     const aboutYourDevice = await OverviewKompaktPage.aboutYourDevice
-    await expect(aboutYourDevice).toBeDisplayed
+    await expect(aboutYourDevice).toBeDisplayed()
     console.log(aboutYourDevice)
     await expect(aboutYourDevice).toBeClickable
     await aboutYourDevice.click()
 
     const sarInformationButtonKompakt =
       await OverviewKompaktPage.sarInformationButtonKompakt
     const sarInformationPopup = await OverviewKompaktPage.sarInformationPopup
-    await expect(sarInformationButtonKompakt).toBeDisplayed
+    await expect(sarInformationButtonKompakt).toBeDisplayed()
     await sarInformationButtonKompakt.click()
-    await expect(sarInformationPopup).toBeDisplayed
+    await expect(sarInformationPopup).toBeDisplayed()
 
     const sarInformationPopupCloseButton =
       await OverviewKompaktPage.sarInformationPopupCloseButton
+    await sarInformationPopupCloseButton.waitForClickable()
     await sarInformationPopupCloseButton.click()
 
+    const backArrowButton = OverviewKompaktPage.backArrowButton
+    await backArrowButton.waitForClickable()
+    await backArrowButton.click()
+
     const createBackupButton = await OverviewPage.createBackupButton
-    await expect(createBackupButton).toBeDisplayed
-    await expect(createBackupButton).toBeClickable
+    await expect(createBackupButton).toBeDisplayed()
+    await expect(createBackupButton).toBeClickable()
 
     const backupInfo = await OverviewKompaktPage.backupInfo
-    await expect(backupInfo).toBeDisplayed
+    await expect(backupInfo).toBeDisplayed()
     const str: string =
       "\tYou haven’t backed up your device yet.\n Create a backup file to protect against data loss.\n"
     const backupInfoText: string = str.trim()
 
     const kompaktBatteryIcon = await OverviewKompaktPage.kompaktBatteryIcon
-    await expect(kompaktBatteryIcon).toBeDisplayed
+    await expect(kompaktBatteryIcon).toBeDisplayed()
 
     const kompaktBatteryLevelValue =
       await OverviewKompaktPage.kompaktBatteryLevelValue
-    await expect(kompaktBatteryLevelValue).toBeDisplayed
+    await expect(kompaktBatteryLevelValue).toBeDisplayed()
     const batteryValNum: string = "100%"
     const batteryValue: string = batteryValNum
```

Comment:
It makes no sense to me why not a one line

```
    const batteryValue: string = "100%"
```

or even assertion with a value? 
```
    await expect(kompaktBatteryLevelValue).toHaveText("100%")
````

Reply:
const kompaktBatteryLevelValue =
      await OverviewKompaktPage.kompaktBatteryLevelValue
    await expect(kompaktBatteryLevelValue).toBeDisplayed()
    await expect(kompaktBatteryLevelValue).toHaveText("100%")